### PR TITLE
Improve and translate extension template

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#492

### CODE REVIEW

File: openfisca_extension_template/local_benefit.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_country_template.entities import *
+
+
+class paris_child_allowance(Variable):
+    column = FloatCol
+    entity = Household
+    definition_period = MONTH
+    label = u"Local benefit : a fixed amount by child each month"
```
Comment (MattiSG) [inline openfisca_extension_template/local_benefit.py]: No spaces before colon in English.

File: openfisca_extension_template/local_benefit.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_country_template.entities import *
+
+
+class paris_child_allowance(Variable):
```
Comment (MattiSG) [inline openfisca_extension_template/local_benefit.py]: Either rename “paris” to “local_town”, or add Paris to the label.

File: openfisca_extension_template/parameters.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-<NODE code="ma_collectivite" description="Aides de ma collectivité">
-  <NODE code="mon_aide" description="Ma première aide">
-    <CODE code="montant" description="Montant de l'aide" format="float" type="monetary">
-      <VALUE deb="2014-01-01" fin="2018-12-31" valeur="100" />
+<NODE code="paris" description="Aides de ma collectivité">
+  <NODE code="child_allowance" description="Ma première aide">
```
Comment (MattiSG) [inline openfisca_extension_template/parameters.xml]: Not translated.

File: setup.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 from setuptools import setup, find_packages
 
 setup(
-    name="Openfisca-MA_COLLECTIVITE",
+    name="OpenFisca-Extension-Template",
     version="1.0.0",
     description="Plugin OpenFisca pour les aides sociales de ma collectivité",
```
Comment (MattiSG) [inline setup.py]: Not translated (`author` as well).

**Comment by fpagnoux** [comment]
Thanks for the fixes !

**Comment by fpagnoux** [comment]
Published

**Pull request merged**